rtos/rtx/cmsis_os.h

```diff
@@ -433,23 +433,23 @@ uint32_t osKernelSysTick (void);
 /// \param         instances     number of possible thread instances.
 /// \param         stacksz       stack size (in bytes) requirements for the thread function.
 #if defined (osObjectsExternal)  // object is external
-#define osThreadDef(name, priority, instances, stacksz) \
+#define osThreadDef(name, priority, stacksz) \
```

Comment:
It is backward compatible. It was removed because our old RTX4 is not up to date with the upstream RTX4, so I had to make the modifications to keep the compatibility.